File: URI/uri-1848/cpp-uri-1848/counting_crow.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
	for(int scream = 0; scream < 3; scream++){
		string 	input 	= "";
		int 	sum 	= 0;
		getline(cin, input);
		while(input.at(0) != 'c'){
			for(int posi = 0; posi < 3; posi++){
				if(input.at(posi) == '*'){
					sum += 1 << 2-posi;
				}
			}
			cin >> input;
		}
		cout << sum << "\n";
	}
 
	return 0;
}

File: URI/uri-1848/python-uri-1848/counting_crow.py
for scream in range(3):
    line = input()
    summ = 0
    while(line[0] != "c"):
        for i in range(3):
            if(line[i] == "*"):
                summ += 1 << (2 - i)
        line = input()
    print(summ);
